server/index.js

```diff
@@ -15,10 +15,58 @@ app.use(morgan('dev'));
 
 app.use(express.static(PUBLIC_DIR));
 
-app.get('/', (req, res) => {
-  res.send('hello from server');
+app.get('/qa/questions/:productId', (req, res) => {
+  const options = {
+    headers: {
+      Authorization: config.TOKEN,
+    },
+  };
+
+  const { productId } = req.params;
+
+  axios.get(`https://app-hrsei-api.herokuapp.com/api/fec2/hr-sea/qa/questions/?product_id=${productId}`, options)
+    .then(({ data }) => {
+      res.status(200).send(data);
+    }).catch((err) => {
+      console.log('there was an error getting questions based on product id', err);
+    });
+});
+
+app.post('/qa/questions/:productId', (req, res) => {
+  const options = {
+    headers: {
+      Authorization: config.TOKEN,
+    },
+  };
+
+  const { productId } = req.params;
+
+  axios.post(`https://app-hrsei-api.herokuapp.com/api/fec2/hr-sea/qa/questions/?product_id=${productId}`, req.body, options)
+    .then(() => {
+      res.status(201).end('created');
+    }).catch((err) => {
+      console.log('error sending post request', err);
+    });
 });
 
+app.get('/qa/questions/:questionId/answers', (req, res) => {
+  const options = {
+    headers: {
+      Authorization: config.TOKEN,
+    },
+  };
+
+  const { questionId } = req.params;
+
+  axios.get(`https://app-hrsei-api.herokuapp.com/api/fec2/hr-sea/qa/questions/${questionId}/answers`, options)
+    .then(({ data }) => {
+      res.status(200).send(data);
+    }).catch((err) => {
+      console.log('there was an error at the server getting answers', err);
+    });
+});
+
+<<<<<<< HEAD
```

Comment:
@EricMHernandez not sure if this is an artifact from when you were resolving merge conflicts?